src/TimeWidget/index.js

```diff
@@ -109,12 +109,13 @@ export default class TimeWidget extends React.Component {
             value={minute}
           />
           <div className="usa-datefield usa-form-group time-period-select-container">
+            <label className="hidden-content" htmlFor={`${id}_time_of_day`}>Time of Day</label>
             <select
-              className="usa-input-inline" name={`${id}am_pm_period`} id={`${id}am_pm_period`}
+              className="usa-input-inline" name={`${id}_time_of_day`} id={`${id}_time_of_day`}
               onChange={event => this.handlePeriodChange(event.target.value)}
             >
-              <option value="AM">AM</option>
-              <option value="PM">PM</option>
+              <option value="am">am</option>
```

Comment:
content has a certain style convention for AM/PM. I forget what it is, but did you by any chance check with them?

Reply:
Yeah! They made a new issue for it: https://github.com/cityofaustin/techstack/issues/1511